pkgs/by-name/sy/synapse-admin-etkecc/package.nix

```
@@ -0,0 +1,74 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  yarnConfigHook,
+  yarnBuildHook,
+  nodejs,
+  nix-update-script,
+  writers,
+}:
```

```suggestion
  baseUrl ? null,
}:
```
please add this parameter and an assertion that fails when it's something else then null

error message should be something like `Please use .withConfig` instead

as this was the old way to configure this

---

[Applied your suggestion](https://github.com/NixOS/nixpkgs/compare/d857cfb955382d92019385174a42b4b9daae9f87..2e18b386b09c1845b1b8036c0b43508be018cc04), however I don't really like how the new formatter indents the entire `mkDerivation` block. Is there any more elegant way to do this? The only way to throw an error I know of that doesn't require this indentation is the `assert e1; e2` syntax, but this doesn't allow setting a custom error message.

---

Thanks!